subprojects/docs/src/docs/userguide/more_about_tasks.adoc

```diff
@@ -61,7 +61,7 @@ We have already seen how to define tasks using strings for task names in <<tutor
 
 [NOTE]
 ====
-This API is likely to be replaced in the near future. See the <<task_configuration_avoidance#sec:what_is_new_task_configuration,task configuration avoidance chapter>> for more information.
+The task configuration APIs are described in more detail in the <<task_configuration_avoidance,task configuration avoidance chapter>>.
 ====
 
 .Defining tasks using strings for task names
```

Comment:
❌ We should [revert sample `defineUsingStringTaskNames`](https://github.com/gradle/gradle/blob/master/subprojects/docs/src/samples/userguide/tasks/defineUsingStringTaskNames/kotlin/build.gradle.kts) for Kotlin DSL.